### PR TITLE
Add turbo prompts

### PR DESCRIPTION
**Pull request by jwehrmann**

### CODE REVIEW

File: prompts/turbo.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    def add_system_message(self, **kwargs):
+        self._add_prompt("system", self.system_prompt.build(**kwargs))
+
+    def add_assistant_message(self, **kwargs):
+        self._add_prompt("assistant", self.assistant_prompt.build(**kwargs))
+
+    def build(self) -> Dict[str, str]:
+        return [{prompt["type"]: prompt["prompt"]} for prompt in self.prompts]
+
+    def _add_prompt(self, prompt_type: str, prompt: Optional[Union[str, DynamicPrompt]] = None):
```
Comment (joaovbsevero) [inline prompts/turbo.py]: Just remove the optional if you are going to raise an exception if it is not provided

**Review by joaovbsevero — CHANGES_REQUESTED** [review]

File: prompts/turbo.py (hunk 1 above)
Comment (jwehrmann) [reply prompts/turbo.py]: Okay. Ask chat wingman about this. :sweat_smile:

**Review by dougsouza — COMMENTED** [review]
Shouldn't we output the same format expected by openai?

```python
import openai

openai.ChatCompletion.create(
  model="gpt-3.5-turbo",
  messages=[
        {"role": "system", "content": "You are a helpful assistant."},
        {"role": "user", "content": "Who won the world series in 2020?"},
        {"role": "assistant", "content": "The Los Angeles Dodgers won the World Series in 2020."},
        {"role": "user", "content": "Where was it played?"}
    ]
)
```

**Comment by jwehrmann** [comment]
@dougsouza yes, fixed it

**Pull request merged**